compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/PrefetchSharedMemoryCopy.cpp

```diff
@@ -113,6 +114,7 @@ class LoopPrefetcher {
     emitBarrier(loc, rewriter);
     emitCompute(mapping[0], rewriter, indVar);
     emitBarrier(loc, rewriter);
+    emitSchedBarrier(loc, rewriter);
```

Comment:
Re @kuhar's point, I'd stick some sort of condition around this to ensure it only runs on ROCm targets?

Reply:
Will do!